src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -174,6 +188,15 @@ public boolean isLogoutLinkVisible() {
     }
   }
 
+  public boolean isExploreWikiHeaderVisible() {
+    try {
+      return exploreWikiHeader.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)